### ISSUE TITLE
Don't submit wrapper pull requests to WS Repos for every PR on this repo

### ISSUE THREAD

**Issue by brentru** [adafruit/Wippersnapper_Protobuf#122]
This (https://github.com/adafruit/Wippersnapper_Protobuf/pull/121#pullrequestreview-1192348381) should not have submitted a pull request. We may want to update the workflow to only submit wrapper pull requests on release, and create a new release for it. 

Not sure how to parse it out otherwise, suppose we could check if the PR contains .proto files.

@lorennorman  - thoughts?

**Comment by lorennorman** [comment]
Yep we can check if .proto files were modified. I do similar stuff in the components and boards repos to make sure only certain files are modified, it starts with [the `gather-relevant-files` job](https://github.com/adafruit/Wippersnapper_Components/blob/main/.github/workflows/validate.yml#L30) and the files list gets input into later steps. So we have a jumping off point when we're ready to fix this.
